import "libnutrition/proto/data/specialized_unit.proto";
import "libnutrition/proto/service/error.proto";

message SpecializedUnitIdentifier 
{
	optional int32 singleFoodId = 1;
	optional int32 sequence = 2;
}

message SpecializedUnitLoadResponse
{
	repeated SpecializedUnitData specializedUnits = 1;
	optional Error error = 100;
}

message SpecializedUnitLoadRequest
{
	optional bool omit = 1 [default = false];
	
	repeated int32 requestedSingleFoodIds = 2;		
	repeated SpecializedUnitIdentifier requestedIds = 3;
}
